dm/loader/lightning.go

```diff
@@ -24,6 +24,9 @@ import (
 	"github.com/pingcap/tidb/br/pkg/lightning"
 	"github.com/pingcap/tidb/br/pkg/lightning/checkpoints"
 	lcfg "github.com/pingcap/tidb/br/pkg/lightning/config"
+	promutil2 "github.com/pingcap/tidb/util/promutil"
+	"github.com/pingcap/tiflow/engine/pkg/promutil"
```

Comment:
I prefer a better name for `promutil2` and `promutil`. It is easy to confuse the two. (It's not a very strong request)

Reply:
will rename later